Detectors/AOD/src/AODMcProducerWorkflowSpec.cxx

```diff
@@ -80,7 +72,7 @@ void AODMcProducerWorkflowDPL::fillMCParticlesTable(o2::steer::MCKinematicsReade
         }
       }
       // enumerate saved mc particles and their relatives to get mother/daughter relations
-      for (int particle = 0; particle < mcParticles.size(); particle++) {
+      for (auto particle = 0U; particle < mcParticles.size(); ++particle) {
```

Comment:
I don't mind the ++ vs ++ but doesn't the compiler optimize this away anyway when you do not use the lvalue?